src/main/scala/outwatch/dom/helpers/DomUtils.scala

```diff
@@ -56,33 +55,26 @@ object Children {
 
     override def ::(node: ChildVNode): Children = node match {
       case s: StringVNode => toModifier(s) :: this // this should never happen
-      case n => n :: VNodes(modifiers.map(toVNode), StreamStatus())
+      case n => n :: VNodes(modifiers.map(toVNode))
     }
   }
 
-  private[outwatch] case class VNodes(nodes: List[ChildVNode], streamStatus: StreamStatus) extends Children {
+  private[outwatch] case class VNodes(nodes: List[ChildVNode], hasStream: Boolean = false) extends Children {
```

Comment:
I would remove the default argument here, because you can easily create an invalid state here, so you should think about this argument when constructung this VNodes class.